src/index.js

```diff
@@ -50,15 +50,15 @@ class WarmUP {
    * */
   afterPackageInitialize () {
     // See https://github.com/serverless/serverless/issues/2631
-    this.options.stage  = this.options.stage
-      || this.serverless.service.provider.stage
-      || (this.serverless.service.defaults && this.serverless.service.defaults.stage)
-      || 'dev'
-    this.options.region = this.options.region
-      || this.serverless.service.provider.region
-      || (this.serverless.service.defaults && this.serverless.service.defaults.region)
-      || 'us-east-1'
-      
+    this.options.stage = this.options.stage ||
```

Comment:
These changes were not necessary and are totally unrelated to this PR.

Please revert them :)

Reply:
linting failed so had to run the other script to fix